SPECS-CROSS/binutils_cross/binutils_cross.spec

```diff
@@ -0,0 +1,248 @@
+# Globals which should be in a macro file.
```

Comment:
Very general question: how and when are these packages built? I'm curious, if we need the two new spec folder now or maybe we can keep the new specs with the core ones.

Reply:
I think the problem with combining these specs with the core specs is that it may not be obvious that these are "cross" packages that are solely meant to be used to build the toolchain that builds the cross-compiler package `cross-gcc`. The packages in `SPECS-CROSS` are built sequentially due to certain dependencies to eventually build a cross compiler (cross-gcc).

Ideally, these would be built through a pipeline and a user that wants to cross-compile would be able to pull the gcc-cross package from PMC and cross-compile source code.